src/game_interpreter_map.cpp

```diff
@@ -1514,6 +1520,8 @@ bool Game_Interpreter_Map::CommandKeyInputProc(RPG::EventCommand const& com) { /
 	int result = 0;
 	size_t param_size = com.parameters.size();
 
+	bool (*check)(Input::InputButton) = wait ? Input::IsTriggered : Input::IsPressed;
```

Comment:
Is this some kind of inline-function declaration? Works with C++03?
Have never seen this before.